buildscripts/draft-change-log-entries.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+
+git log --reverse --pretty=format:"- %s" "$1"..HEAD \
+  | sed -r 's,\(#([0-9]+)\),\n  ([#\1](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/\1)),'
```

Comment:
```suggestion
  | sed -r 's,\(#([0-9]+)\),([#\1](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/\1)),'
```

As discussed on Slack, maybe it's simplest to not add this newline since rendered MD will be fine. We could probably remove the other script then since it'd be a 1:1 copy-pasta

Reply:
> We could probably remove the other script then since it'd be a 1:1 copy-pasta

ya, that's a good point, will do 👍